UI/homepage.py

```diff
@@ -33,13 +96,30 @@ def create_home_screen():
 
 # Screen state
 current_screen = 'home'
-input_box = search_results = back_button = filter_title = recipe_title = items_text_box = None
+input_box = search_results = back_button = filter_title = recipe_title = recipe_search_button = items_text_box = None
 item_list = []
 delete_buttons = []
 
+# Plus Button
+plus_button = None
+
+# Minus Button
+minus_button = None
+
+# Value
+value = 0
+
+# Value Label
+value_label = None
+
+# Checkboxes
+check_boxes = []
+
+filter_buttons = []
+
 # Handling screens and transitions
 def draw_filter_search_screen():
-    global input_box, search_results, back_button, filter_title, items_text_box, current_screen, delete_buttons
+    global input_box, search_results, back_button, filter_title, items_text_box, current_screen, plus_button, minus_button
```

Comment:
why make this a global variable instead of a variable tied to self? (ex: `self.x`)